parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -176,7 +176,6 @@ configureSwapFile
 ensureSysctl
 ensureKubelet
 ensureJournal
-ensureUpdateNodeLabels
```

Comment:
what about those scripts?

Reply:
The reason I didn't do a revert of the previous change was due to the fact that I took care of the other label service, and didn't want it popping back up again

Reply:
[10:17 AM] Charlie McBride
    The reason I didn't do a revert of the previous change was due to the fact that I took care of the other label service, and didn't want it popping back up again
(1 liked)

Reply:
Added an issue to track code clean up for the service:
https://github.com/Azure/AgentBaker/issues/684

Reply:
We can simply remove those service files to reduce garbage code